sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_download_stream.py

```diff
@@ -24,7 +24,7 @@ class AsyncDownloadBlobStream(
     def __init__(
         self,
         *,
-        response: PipelineResponse,
+        response: PipelineResponse[Any, Any],
```

Comment:
Same comment than earlier in this PR, you don't have to use Any here

Reply:
Updated